user/helm-chart/values-kubernets.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Prefix used for defining KubeStellar objects in the cluster
+prefix: kubestellar
+
+# Namespace where KubeStellar will be deployed
+namespace: kubestellar
+
+# KubeStellar image parameters
+image:
+  repository: quay.io/kubestellar/kubestellar
+  pullPolicy: IfNotPresent
+  tag: stable
```

Comment:
Are we committing here to make to externally-relevant changes to that container image after users start using this Helm chart?  I hope not.

Reply:
the stable tag, as in all the registry, will move along to point to different versions